pybossa/wizard.py

```diff
@@ -69,6 +60,15 @@ def task_presenter(self):
 
         return task_presenter
 
+    def task_guidelines(self):
+        if self.not_project_exist():
+            return False
+        guideline_editor_default_value = '<p><br></p>'
```

Comment:
Just curious if there was a reason for using a pre-defined HTML block for initializing or could this be left empty? :)

```python
guideline_editor_default_value = ''
```

Reply:
The editor produce that value  '<p><br></p>' if there is nothing. But actually I should check for '' and '<p><br></p>' assuming they click save with an empty editor.

A better way might be check if the value is equal that '<p><br></p>' then I keep it  as ''. Then I only need that default value at the saving.